src/lib.rs

```diff
@@ -192,37 +182,43 @@ fn get_min_sec_from_duration(duration: time::Duration) -> (u64, u64) {
     (minutes, seconds)
 }
 
-fn show_notification(title: &str, message: &str) {
+// Here it take &Path instead of &PathBuf because the compiler say so. Not sure why :D
+// Also here for Mac users i remove the sound when the no_sound flag is on.
+// But not sure so far if we need to keep the sound or use the sound_play as we did for linux
+fn show_notification(title: &str, message: &str, sound: &Path, no_sound: &bool) {
     if cfg!(target_os = "macos") {
-        match process::Command::new("osascript")
-            .arg("-e")
-            .arg(format!(
-                "display notification \"{}\" with title \"{}\"",
-                message, title
-            ))
-            .arg("-e")
-            .arg(format!("say \"{}\" using \"Thomas\"", message))
-            .output()
-        {
+        let mut cmd = process::Command::new("program");
```

Comment:
Important: we need to revert "program" to "osascript" to get it to work on MacOS.
```suggestion
        let mut cmd = process::Command::new("osascript");
```